test/TTConfigurationData.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+from cp3_llbb.Framework import Framework
+
+process = Framework.create(True, eras.Run2_50ns, '74X_dataRun2_v2', cms.PSet(
+    dilepton = cms.PSet(
+        type = cms.string('dilepton_analyzer'),
+        prefix = cms.string('dilepton_'),
+        enable = cms.bool(True),
+        categories_parameters = cms.PSet(
+            mll_cut = cms.untracked.double(20)
+            ),
+        parameters = cms.PSet(
+            standalone = cms.untracked.bool(True),
```

Comment:
This should be set to False as we don't want the dilepton analyzer to register its own categories.